docs/AboutUs.md

```diff
@@ -1,9 +1,9 @@
 # About us
 
-| Display                                             |       Name        |             Github Profile              |              Portfolio               |
-|-----------------------------------------------------|:-----------------:|:---------------------------------------:|:------------------------------------:|
-| ![](https://via.placeholder.com/100.png?text=Photo) |     Ming Chun     | [Github](https://github.com/mcmc101001) | [Portfolio](docs/team/mcmc101001.md) |
-| ![](https://via.placeholder.com/100.png?text=Photo) |    Shan YuXuan    |      [Github](https://github.com/)      |  [Portfolio](docs/team/johndoe.md)   |
-| ![](https://via.placeholder.com/100.png?text=Photo) | Daniel Pappalardo |      [Github](https://github.com/)      |  [Portfolio](docs/team/johndoe.md)   |
-| ![](https://via.placeholder.com/100.png?text=Photo) |   Benedict Chan   |      [Github](https://github.com/)      |  [Portfolio](docs/team/johndoe.md)   |
-| ![](https://via.placeholder.com/100.png?text=Photo) |     Kai Ning      |      [Github](https://github.com/)      |  [Portfolio](docs/team/johndoe.md)   |
+Display |       Name        | Github Profile | Portfolio 
+--------|:-----------------:|:--------------:|:---------:
+![](https://via.placeholder.com/100.png?text=Photo) |     Ming Chun     | [Github](https://github.com/mcmc101001) | [Portfolio](team/mcmc101001.md)
+![](https://via.placeholder.com/100.png?text=Photo) |    Shan YuXuan    | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
+![](https://via.placeholder.com/100.png?text=Photo) | Daniel Pappalardo | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
+![](https://via.placeholder.com/100.png?text=Photo) |   Benedict Chan   | [Github](https://github.com/BenedictChannn) | [Portfolio](team/benedictchannn.md)
```

Comment:
Maybe we should all also add a profile picture, to make this page look more professional